Documentation/teaching/labs/kernel_api.rst

```diff
@@ -716,26 +716,26 @@ free them (in :c:func:`memory_exit`).
       (use :c:func:`kfree`).
 
 .. hint::
```

Comment:
I think you also need to fix the indentation for this line and the following `note::` and `hint::` in this TODO.

Reply:
No, I indented only the ones related to the TODO above them. These two apply to the whole exercise.

Reply:
Ok, thank you for the explanation.